src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImplTest.java

```diff
@@ -27,27 +28,34 @@
     private ChipsRestClient chipsRestClient;
 
     @Mock
-    private MessageProducer retryMessageProducer;
+    private MessageProducer messageProducer;
 
     @Mock
     private ApplicationLogger logger;
 
     @InjectMocks
     private MessageProcessorServiceImpl messageProcessorService;
 
+    @BeforeEach
+    void Setup(){
```

Comment:
methods should start with lowercase letters

Reply:
.net-ism changed to lowercase.